src/v2/providers/alerts/alerts.ts

```diff
@@ -86,6 +86,11 @@ export interface FirebaseAlertOptions extends options.EventHandlerOptions {
   /** Scope the function to trigger on a specific application. */
   appId?: string;
 
+  /**
+   * If true (or resolving to true), do not deploy this function.
```

Comment:
I think we can leave it as just `If true` - since that's how we phrase any other config value.

Reply:
Done.